docs/operations/annotations_de.md

```diff
@@ -0,0 +1,96 @@
+# Annotationen für den k8s-dogu-operator
+
+Dieses Dokument enthält die detaillierte Beschreibung aller erzeugten Annotationen des k8s-dogu-Operators.
+
+## Dienste
+
+Dieser Abschnitt enthält alle Anmerkungen, die an K8s-Dienste angehängt werden, falls erforderlich.
+
+### k8s-dogu-operator.cloudogu.com/ces-services
+
+Die Annotation `k8s-dogu-operator.cloudogu.com/ces-services` enthält Informationen über einen oder mehrere ces-Dienste.
+Jeder ces-Dienst definiert einen externen Dienst des Systems, der über den Webserver erreichbar ist. Die Annotation wird automatisch
+für jede dogu generiert, die als Webapp markiert ist. Es ist auch möglich, das Verhalten der Dienste anzupassen, d.h. eine
+benutzerdefinierte Url, über die der Dienst erreichbar ist.
```

Comment:
`Es ist auch möglich, das Verhalten der Dienste anzupassen, d.h. eine benutzerdefinierte Url, über die der Dienst erreichbar ist.` Was ist mit dieser URL?  Kann man die einstellen? Wie hat die Einfluss auf das Verhalten des Dienstes?

Reply:
I changed the content a little bit. The URL is a Uniform Resource Locator and is used to identify the service endpoint. This service endpoint is often the same for most of our Dogus. However, some of them need some customization, e.g., adding a slash at the end of the URL.